src/ast/parse.lalrpop

```diff
@@ -60,9 +60,10 @@ Command: Command = {
     LParen "ruleset" <name:Ident> RParen => Command::AddRuleset(name),
     LParen "rule" <body:List<Fact>> <head:List<Action>> <ruleset:(":ruleset" <Ident>)?> <name:(":name" <String>)?> RParen => Command::Rule{ruleset: ruleset.unwrap_or("".into()), name: name.unwrap_or("".to_string()).into(), rule: Rule { head, body }},
     LParen "rewrite" <lhs:Expr> <rhs:Expr>
+        <replace:(":replace")?>
```

Comment:
I think I would prefer to have both `:subsume` and `:unextractable` here.
I think it's more clear, and users don't have to learn that `:replace = :subsume + :unextractable`